lib/src/hydrated_bloc_storage.dart

```diff
@@ -67,4 +67,19 @@ class HydratedBlocStorage implements HydratedStorage {
     _instance = null;
     return await _file.exists() ? await _file.delete() : null;
   }
+
+  static Future<Directory> getDocumentDir({bool testing = false}) async {
+    if (!testing) {
```

Comment:
why is this needed?

Reply:
the testing method? Because it picks up the linux path of the circus CI linux directory and not the temp directory. On desktop when not testing it works great.

Reply:
But with this approach it won't be possible to test the MacOS/Linux or Windows 😢 Can we instead fake the environments in the tests without changing the implementation?

Reply:
Let me see what i can do!

Reply:
@felangel this is the error im getting ` FileSystemException: Cannot open file, path = '/home/cirrus/.config/.hydrated_bloc.json' (OS Error: No such file or directory, errno = 2)` not sure what to do for testing.

Reply:
I'll try to take a look later today and see if I have any luck 🤞

Reply:
OK sweet!

Reply:
I have a fix 😄
Can you add me as a collaborator to your fork?

Reply:
Done! ✅

Reply:
Can you review https://github.com/AppleEducate/hydrated_bloc/pull/1?

Reply:
Merged! 🚀